markdown/opentelemetry-intro/opentelemetry-intro.md

```diff
@@ -0,0 +1,320 @@
+author: Jesse Menning
+summary:
+id: opentelemetry-intro
+tags:
+categories: opentelemetry, nyctaxi
+environments: Web
+status: Draft
+feedback link: <insert github repo or leave empty>
```

Comment:
The feedback link needs to be updated to `https://github.com/jmenning-solace/nycmoderntaxi/tree/master/opentelemetry-intro`